### ISSUE TITLE
Default Value of Listbox Select does not show

### ISSUE THREAD

**Issue by dajoeberlei** [livewire/flux#412]
Default Value of Listbox Select does not show. When removing the `variant='listbox'` it works as intended and the normal select dropdown displays the selected default "EUR".

```php 
@dump($form->currency) // shows "EUR"

<flux:select label="Account Currency" wire:model="form.currency" variant="listbox"> // does not show default "EUR" in the input box
    @foreach(\App\Enums\Currency::asSelectArray() as $key => $currency)
        <flux:option value="{{ $key }}">{{ $currency }}</flux:option> // the "EUR" option has the checkmark though.
    @endforeach
</flux:select>
```

```php
// Form Class
public $currency = Currency::EUR->value;
```

![Image](https://github.com/user-attachments/assets/51c3005c-a72a-4812-bd17-cb7f7205596f)

**Comment by jeffchown** [comment]
@dajoeberlei Can you provide your Currency enum code too?

It works as expected with a regular array:

```
@php
    $currencies = [
            'EUR' => 'EUR',
            'USD' => 'USD',
            'INR' => 'INR',
            'JPY' => 'JPY',
            'GBR' => 'GBR',
    ];
@endphp

<div>
    @dump($form->currency)

    {{-- **does** show default "EUR" in the input box --}}
    <flux:select label="Account Currency" wire:model="form.currency" variant="listbox">
        @foreach($currencies as $key => $currency)
            <flux:option value="{{ $key }}">{{ $currency }}</flux:option>
        @endforeach
    </flux:select>
</div>
```

![Image](https://github.com/user-attachments/assets/3267e9a7-4733-420d-9ce8-4f8fe920a0e3)

**Comment by jeffchown** [comment]
@dajoeberlei Hmmm...

Are you using v1.0.9 of Flux?  

If so, did you delete any Flux components you had published via `php artisan flux:publish` and then republish them (if needed) after updating to v1.0.9?
And, lastly, did you `php artisan view:clear` to clear the view cache after updating Flux? (missing this one caused one of my issues a while back)

**Comment by dajoeberlei** [comment]
@jeffchown that was a good guess! I actually did publish old ones a few weeks ago but i had them deleted already and i tried `php artisan view:clear` again as you suggested. Unfortunately with no success. I published the component again but it didnt help.

I'll close this issue since i have a feeling its a bug that seems to be fixed but i have not managed to apply it yet but probably will find out in the future how to. Its not that important to me because the general select does the trick for me. If I encounter it again at a ill reopen
